mopro-wasm/src/app_config/web.rs

```diff
@@ -0,0 +1,114 @@
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+
+pub fn build() {
+    // The `mopro-wasm` package requires compilation using the nightly version of Rust.
+    // The build method that uses the `wasm-pack` crate cannot be called directly in `test-e2e` as it relies on stable Rust.
+    // Rust does not support mixing different toolchains or targets in a single build, hence this separation.
+    // Note: There is a known another issue with `wasm-pack` that may affect the build process. For more information, see: https://github.com/rustwasm/wasm-pack/issues/1400.
+    let cwd = std::env::current_dir().expect("Failed to get current directory");
+    let bindings_dest = Path::new(&cwd).join("MoproWasmBindings");
+
+    // Check if `wasm-pack` command is available
+    let check_status = Command::new("wasm-pack").arg("--version").status();
+    if check_status.is_err() || !check_status.unwrap().success() {
+        eprintln!("Error: `wasm-pack` command not found. Please install `wasm-pack` to proceed.");
+        std::process::exit(1);
+    }
+
+    // Search the `mopro-wasm` directory
+    let mopro_wasm_dir = if cwd
+        .parent()
+        .map_or(false, |p| p.join("mopro-wasm").exists())
+    {
+        // When running the script from `test-e2e`
+        cwd.parent()
+            .expect("Failed to get parent directory")
+            .join("mopro-wasm")
+    } else {
+        // When running the script from the CLI template or else, download 'mopro-wasm' from repo
+        fetch_mopro_wasm();
+        cwd.join("mopro-wasm")
+    };
```

Comment:
In this case I would suggest that you create a PR for `mopro-wasm/src/app_config/web.rs` first
and then complete the CLI using
```toml
mopro-wasm = { git = "https://github.com/zkmopro/mopro" }
```
so that you don't even have to clone the mopro repo to use `mopro-wasm`